backend/api/user.go

```diff
@@ -53,7 +53,17 @@ func Register(c *gin.Context) {
 		return
 	}
 	user.Overview = c.PostForm("overview")
-	user.Organization = c.PostForm("organization")
+	insIDStr := c.PostForm("institutionID")
+	if insIDStr == "" {
+		user.InstitutionID = 1
```

Comment:
For this default value, can you add it to docs?

Reply:
For now, I choose to add a institution named "no institution", so I will delete it. Then, the field "institutionID" is required.